install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -355,6 +355,9 @@ data:
   enable-external-ips: {{ .Values.global.externalIPs.enabled | quote }}
 {{- end }}
 {{- end }}
+{{- if (or (eq .Values.global.kubeProxyReplacement "strict") (eq .Values.global.kubeProxyReplacement "probe")) }}
```

Comment:
I think it's enough to check only for "disabled".

Reply:
How do I check that?

Reply:
Done.